### ISSUE TITLE
Reorganise menu

### ISSUE THREAD

**Issue by david-r-edgar** [projectwife/mtesitoo-android#112]
In the menu, it's not clear how to view the list of products. In fact, "Home" is the list of products. This is not clear.

**We should rename "Home".**

The menu is badly organised. eg. 'Orders' is under the account sub-heading.

Probably **we should reorganise the menu** something like this:

```  Vendor name
    Seller
-------------
Products
+ New Product
Orders
-------------
==Account==
Profile
-------------
==App==
Info
Help
Contact
Logout
```

**Comment by ptyagicodecamp** [comment]
fixed in PR: https://github.com/projectwife/mtesitoo-android/pull/130

**Comment by ptyagicodecamp** [comment]
fix deployed to play store (vercode #8).
